modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -0,0 +1,9 @@
+[id="creating-a-new-client-in-keycloak_{context}"]
+= Creating a new client in Keycloak
+
+Create a new client in Keycloak for secure and easy identity and access management. Keycloak acts as an Identity broker between ArgoCD and OpenShift, as a result you can login into Argo CD using the OpenShift credentials also.
```

Comment:
minor grammatical changes. what do you think about this rewrite:

`Create a new client in Keycloak for security, easy identity, and access management. Keycloak acts as an identity broker between ArgoCD and OpenShift; as a result, you can login into Argo CD using the OpenShift credentials.` 

Note, I looked up `identify broker` in other RH documentation and they don't capitalize See this page for an example: https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.0/html/server_administration_guide/identity_broker